pkg/vm/vm.go

```diff
@@ -1055,6 +1055,11 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 				panic("zero modulus") // https://docs.microsoft.com/en-us/dotnet/api/system.numerics.biginteger.modpow?view=net-6.0#exceptions
 			}
 			res.Exp(base, exponent, modulus)
+
+			if base.Sign() < 0 && exponent.Bit(0) == 1 {
```

Comment:
`exponent.Bit(0) == 1` defines if `exponent` is even. Is it work the same way on LE/BE arch?

Reply:
it doesn't depend on the arch. ` Bit(0)` is about logical representation not physical in memory. 
and:
```
// Bit returns the value of the i'th bit of x. That is, it
// returns (x>>i)&1. The bit index i must be >= 0.
func (x *Int) Bit(i int) uint {
	if i == 0 {
		// optimization for common case: odd/even test of x
		if len(x.abs) > 0 {
			return uint(x.abs[0] & 1) // bit 0 is same for -x
		}
		return 0
	}
```

Reply:
Good.